app/controllers/users_controller.rb

```diff
@@ -1,4 +1,24 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
[IMO] Use find instead of `find_by`

Reply:
[IMO] when we use find, the function may return an exception (not found) that we have to handle it. In contract, find_by return a nil value with out exception.

Reply:
@minhthuc It is ok because I will raise ActiveRecord::RecordNotFound => 404 https://makandracards.com/makandra/41790-how-rails-chooses-error-pages-404-500-for-exceptions. We dont need to care about this case. (User tries to manually access via putting ID in URL is not a normal case)